CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
```

Comment:
Hm, difficult (FYI: although it may not seem like, I already cleaned up a lot 😄). The PR was much more delicate than I originally anticipated and the remaining commits are pretty much self-contained I think... Anything specific you'd like me to change?

Reply:
Because I'm trying to push the PR towards merge and this is rather minor, I'll resolve. But if you want me to change something, please feel free to reopen

Reply:
Had a brief look and I think this looks good 👍 My original comment was due to the many commits "Fix XYZ" because I thought that these mean to fix some error you introduced during the PR, but that was just me misunderstanding your use of that word